NuRadioMC/examples/05_pulser_calibration_measurement/radioEmitting_pulser_calibration/runARA02.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+import h5py
+import numpy as np
+import datetime
+import NuRadioReco.modules.triggerTimeAdjuster
+import NuRadioReco.modules.ARIANNA.hardwareResponseIncorporator
+import NuRadioReco.modules.efieldToVoltageConverter
+import NuRadioReco.modules.voltageToEfieldConverter
+import NuRadioReco.modules.trigger.simpleThreshold
+import NuRadioReco.modules.trigger.highLowThreshold
+import NuRadioReco.modules.trigger.powerIntegration
+import NuRadioReco.modules.channelResampler
+import NuRadioReco.modules.channelBandPassFilter
+import NuRadioReco.modules.channelGenericNoiseAdder
+import NuRadioReco.modules.custom.deltaT.calculateAmplitudePerRaySolution
+from NuRadioReco.utilities import units
+from NuRadioMC.simulation import simulation as simulation
+from NuRadioReco.framework.parameters import channelParameters as chp
+from NuRadioReco.framework.trigger import Trigger
+import logging
+logging.basicConfig(level=logging.WARNING)
+logger = logging.getLogger("runARA02")
+efieldToVoltageConverter = NuRadioReco.modules.efieldToVoltageConverter.efieldToVoltageConverter()
+efieldToVoltageConverter.begin()
+simpleThreshold = NuRadioReco.modules.trigger.simpleThreshold.triggerSimulator()
+powerIntegration = NuRadioReco.modules.trigger.powerIntegration.triggerSimulator()
+highLowThreshold = NuRadioReco.modules.trigger.highLowThreshold.triggerSimulator()
+channelResampler = NuRadioReco.modules.channelResampler.channelResampler()
+channelBandPassFilter = NuRadioReco.modules.channelBandPassFilter.channelBandPassFilter()
+channelGenericNoiseAdder = NuRadioReco.modules.channelGenericNoiseAdder.channelGenericNoiseAdder()
+electricFieldResampler = NuRadioReco.modules.electricFieldResampler.electricFieldResampler()
+calculateAmplitudePerRaySolution = NuRadioReco.modules.custom.deltaT.calculateAmplitudePerRaySolution.calculateAmplitudePerRaySolution()
+triggerSimulator = NuRadioReco.modules.trigger.highLowThreshold.triggerSimulator()
+triggerTimeAdjuster = NuRadioReco.modules.triggerTimeAdjuster.triggerTimeAdjuster()
+triggerSimulator.begin(log_level=logging.WARNING)
+class mySimulation(simulation.simulation):
+
+    def _detector_simulation_filter_amp(self, evt, station, det):
+        # bandpass filter trace, the upper bound is higher then the sampling rate which makes it just a highpass filter
+         channelBandPassFilter.run(evt, station, det, passband=[80 * units.MHz, 1000 * units.GHz],
+                                  filter_type='butter', order=2)
+         channelBandPassFilter.run(evt, station, det, passband=[0, 500 * units.MHz],
+                                  filter_type='butter', order=10)
+
+ 
+    def _detector_simulation_trigger(self, evt, station, det):
+        # save the amplitudes to output hdf5 file
+        # save amplitudes per ray tracing solution to hdf5 data output
+        calculateAmplitudePerRaySolution.run(self._evt, self._station, self._det)
+        triggerSimulator.run(evt, station, det,
```

Comment:
you can give the triggers a name, that might be useful to later better see which one triggered.